recipes/bootstrap_contrast/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "bootstrap_contrast" %}
+{% set version = "0.321.1" %}
+{% set sha256 = "42c189d5fbf7bb8c672279154b5f6bf392ea7149b1862f1c6cab0998fc06f504" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - matplotlib
+    - seaborn
+    - pandas
+    - numpy
+    - scipy
+
+test:
+  imports:
+    - bootstrap_contrast
+
+about:
+  home: http://github.com/josesho/bootstrap_contrast
+  license: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
If you have the time please report the lack of license upstream. It is quite bad that we need to fix a source distro bug like that :-(